.github/workflows/container-ci.yaml

```diff
@@ -196,7 +208,7 @@ jobs:
         username: ${{ secrets.REGISTRY_USER }}
         password: ${{ secrets.REGISTRY_TOKEN }}
     - name: Test Container Group
-      uses: intel/ai-containers/test-runner@main
+      uses: intel/ai-containers/test-runner@leminh/test-runners
```

Comment:
Same here

Reply:
Will make the changes after I have done revalidating.